instrumentation/spark-2.3/README.md

```diff
@@ -0,0 +1,10 @@
+# Spark Instrumentation
+
+This instrumentation is for
+[Spark](https://github.com/perwendel/spark) -
+a tiny web framework for Java. It is also sometimes referred to as
+"Spark Java" to differentiate it from Apache Spark.
+
+This instrumentation is NOT for Apache Spark.
```

Comment:
seems pretty well covered above already?
```suggestion
```

Reply:
I wanted it to be very clear. :)